framework/Sources/VGSFramework/Core/Enums.swift

```diff
@@ -62,6 +62,8 @@ internal extension FieldType {
             return VGSTextField.cvcRegexForAnyCardType
         case .expDate:
             return "^(0[1-9]|1[0-2])\\/?([0-9]{4}|[0-9]{2})$"
+        case .cardHolderName:
+            return "^([a-zA-Z0-9\\ \\,\\.\\-\\']{2,})$"
```

Comment:
Hard to understand this Regex ☹️
I would like to set this one 
`^([\w]{2,})+([\s]{1})+([\w]{2,})$`
(min any 2 symbols) + (one space) + (min any 2 symbols)

Reply:
Full Name can have more then 2 names. Also in some countries it's allowed to have only 1 name